_posts/2020-11-25-#19858.md

```diff
@@ -10,13 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+- An [eclipse attack](https://bitcoinops.org/en/topics/eclipse-attacks/) is
+  when an adversary is able to isolate a victim from the bitcoin network by
```

Comment:
s/bitcoin/Bitcoin/

can also link to https://bitcoincore.reviews/16702#eclipse-attack

Reply:
Done!